Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -105,12 +105,13 @@ class PlaceDetailsCardViewController: UIViewController {
         setLocation(location: locationProvider?.getCurrentLocation())
         setupCardInteractions()
 
-        pageControl.numberOfPages = place.photoURLs?.count ?? 0
+        pageControl.numberOfPages = place.photoURLs.count
     }
 
     private func setLocation(location: CLLocation?) {
         if let lastTravelTimes = place.lastTravelTime {
             self.cardView.updateTravelTimesUI(travelTimes: lastTravelTimes)
+            return
```

Comment:
> It is possible that the travel times we have cached are now out of date. That is why I left to to display the cached times and then update when the new travel times have been fetched.

I'm concerned about showing out-of-date travel times that may arbitrarily update while the user is looking at them. I'd suggest we stop caching but...

> If I did not do it this way, then it was possible that when scrolling quickly through the cards all the eta's would be spinners.

This is an issue that UX would probably agree is important.

> What if the user has moved significantly since they last viewed this card.

The [location manager has functionality](https://developer.apple.com/reference/corelocation/cllocationmanager/1423500-distancefilter) to only update with location events if they exceed a certain distance. (keeping the early return) we could set this filter and, when we receive an event, we could null all cached travel time values, reducing redundant computation (from removing the return value) and providing a better UX.

But then again, it's more complicated to implement so perhaps it's not important enough.